tools/observability/grafana-prom-values.yaml

```diff
@@ -43,3 +43,7 @@ prometheus:
         - source_labels: [__meta_kubernetes_pod_node_name]
           action: replace
           target_label: kubernetes_node
+
+kube-state-metrics:
+    metricLabelsAllowlist:
+      - pods=[*]
```

Comment:
Updated to list out labels; the performance implications come in when there's a lot of labels ... I was thinking it would be less brittle to allow all but better safe than sorry when it comes to performance.